src/code/qualification/generateOutputFolders.m

```diff
@@ -0,0 +1,66 @@
+function  [SectionsOut, Id] = generateOutputFolders(SectionsIn, REOutputFolder)
+%GENERATEOUTPUTFOLDERS Support function: 
+% Explore tree of sections and generate folders of sections within REOutputFolder 
+% linearize the section structure
+%
+%   generateOutputFolders(SectionsIn, REOutputFolder)
+%       SectionsIn (structure or cell) sections to be created
+%       REOutputFolder (string) name of folder where the sections
+%
+
+% Open Systems Pharmacology Suite;  http://open-systems-pharmacology.org
+
+%% ---------------------------------------------------% 
+
+% Check for Sections Class, 
+% if cell: multiple sections, if struct, only one available
+if iscell(SectionsIn)    
+
+% Loop on every section
+for i = 1:length(SectionsIn)

```

Comment:
Indentation problem: the for loop should be indented (realtive to `if iscell(SectionsIn)`)